lib/influxdb/query/core.rb

```diff
@@ -67,9 +69,16 @@ def write(data, precision, retention_policy = nil)
       def query_params(query, opts)
         precision   = opts.fetch(:precision, config.time_precision)
         epoch       = opts.fetch(:epoch, config.epoch)
+        chunk_size  = opts.fetch(:chunk_size, config.chunk_size)
 
         params = { q: query, db: config.database, precision: precision }
         params[:epoch] = epoch if epoch
+
+        if chunk_size
+          params[:chunked] = 'true'
+          params[:chunk_size] = chunk_size.to_s
```

Comment:
Removed to_s. Changed tests to set an integer as the configuration variable. Forcing it with .to_i appears to be unnecessary at this stage (a string could actually work) so no further changes.